spec/Section 2 -- Language.md

```diff
@@ -1094,3 +1095,24 @@ and operations.
 
 As future versions of GraphQL adopt new configurable execution capabilities,
 they may be exposed via directives.
+
+
+## Document Directives
+
+DocumentDirective :
+  - Directive[Const]
+
+{DocumentDirective} is special kind of {Definition} and should be handled
+on AST level by implementation in similar way like {TypeSystemDefinition}
+or {ExecutableDefinition}.
+
+Documents of any kind may contain zero or more {DocumentDirective}s.
+Same directive may be repeated multiple times at document level:
+
+```graphql example
+@import(from: "./common")
+@import(from: "./user")
```

Comment:
I don't think we should use `@import` directives as an example here.
Since some readers can think it a standard one and supported by GraphQL spec.
I think it would be better to use something like `@someTopLevelDirective`, `@foo`, `@test`, etc.

Reply:
This is most relevant example I wish to show and indeed it is my intent to push on some common way to do import. I can add note or warning below that import is not built-in

Reply:
As `@import` will have meaning in a bunch of libraries almost immediately, and those meaning will probably immediately diverge, let's keep it out of the spec entirely. There aren't any descriptions of any other user-level directives in the spec.

The only other usage is `@addedDirective` and `@example` because they illustrate the usage without introducing a new concept.

In this case, you could call it `@exampleDocumentDirective` or even just `@example`